runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -69,9 +69,6 @@ Java_sun_misc_Unsafe_defineAnonymousClass(JNIEnv *env, jobject receiver, jclass
 	J9JavaVM *vm = currentThread->javaVM;
 	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
 
-	/* For JSR335 this should be NULL */
-	Assert_JCL_true(constPatches == NULL);
```

Comment:
We've been asserting here as we don't implement cp patching yet.  Can you confirm whether the VarHandle code needs CP patching?  And the types of CP entries needing to be patched?